tests/integrations/mcs/resourcemanager/resource_manager_test.go

```diff
@@ -402,8 +403,9 @@ func (suite *resourceManagerClientTestSuite) TestResourceGroupController() {
 		CPUMsCost:        1,
 	}
 
-	controller, _ := controller.NewResourceGroupController(suite.ctx, 1, cli, cfg, controller.EnableSingleGroupByKeyspace())
```

Comment:
why remove `EnableSingleGroupByKeyspace`?

Reply:
Enabling it will prevent the controller from starting the watch channel, this test needs it.